src/cmake/testing.cmake

```diff
@@ -111,6 +111,20 @@ macro (oiio_add_tests)
                              "OIIO_TESTSUITE_SRC=${_testsrcdir}"
                              "OIIO_TESTSUITE_CUR=${_testdir}"
                              ${_ats_ENVIRONMENT})
+            if (NOT DEFINED ENV{GITHUB_ACTIONS})
+              # the github actions run all the tests from the dist tree
+              # and do an install prior to running the tests so skip
+              # setting these here
+              if (NOT DEFINED ENV{OpenImageIO_ROOT})
+                # this will ensure the appropriate fonts are found
+                set_property(TEST ${_testname} APPEND PROPERTY ENVIRONMENT
+                  "OpenImageIO_ROOT=${CMAKE_SOURCE_DIR}/src")
+              endif()
```

Comment:
I'm not 100% sure I understand all the implications -- now or after future changes we aren't considering -- of setting OpenImageIO_ROOT to point to the src directory, of all places. 

And what happens if somebody is building in an environment where OpenImageIO_ROOT is already set to someplace other than this build area (maybe to an existing install)? In that case, this would not be set, and what would that mean?

Wouldn't it be more direct to set OPENIMAGEIO_FONTS to exactly the right spot? And much less likely to affect absolutely anything else?

Also, sorry about this nit, but would you mind using 4-space indentation like the surrounding code? Thanks.